### ISSUE TITLE
Child relationship field not parsing (5.2.3)

### ISSUE THREAD

**Issue by helmi-pts** [ExpressionEngine/ExpressionEngine#229]
**Description of the problem**
Updating from 5.1.3
Child relationship tag does not parse if there is another child relationship inside its tag.

**How To Reproduce**
Channel: Booklist
Relationship Field: rel_books (related to Channel Books)

Channel: Books
Relationship Field: bk_author (related to Channel Authors)

Channel: Authors

Code:
```
{exp:channel:entries channel="booklist" entry_id="9362" rdf="off" limit="1"}
   {rel_books limit="30"}
      {rel_books:bk_author}{rel_books:bk_author:title}{/rel_books:bk_author}
   {/rel_books}
{/exp:channel:entries}
```

--Update--
If you just use the child tag without any parameters; it works as intended.
<code>{rel_books}</code>

This code will not parse:
<code>{rel_books limit="30"}</code>

**Error Messages**
This line is displayed in the browser

<code>{rel_books limit="30" channel="books"} {rel_books:bk_author}{rel_books:bk_author:title}{/rel_books:bk_author} {/rel_books} </code>

**Environment Details:**
 - Version: 5.2.3
 - PHP Version 7.1
 - MySQL Version 5.6
 - OS: Linux
 - Web Server: Apache

**Comment by skookie000** [comment]
Was able to replicate this.  +1

**Comment by dareiff** [comment]
I believe I'm having the same issue on the update to 5.2.3. +1

**Comment by robinsowell** [comment]
Can you guys try the fix in this commit: https://github.com/ExpressionEngine/ExpressionEngine/commit/82aeb671619290ce7ab7ade43cf482aabd823aaa

**Comment by dareiff** [comment]
> Can you guys try the fix in this commit: [82aeb67](https://github.com/ExpressionEngine/ExpressionEngine/commit/82aeb671619290ce7ab7ade43cf482aabd823aaa)

Absolutely it does — amazing, thanks!

**Comment by akolinski** [comment]
I was able to replicate this. Once I removed the parameters from the tag it worked also.
